doc/md_docs.go

```diff
@@ -72,6 +72,17 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 		buf.WriteString(fmt.Sprintf("```\n%s\n```\n\n", cmd.UseLine()))
 	}
 
+	if len(cmd.Aliases) > 0 {
+		buf.WriteString("Aliases: ")
+		for i, alias := range cmd.Aliases {
+			if i > 0 {
+				buf.WriteString(", ")
+			}
+			buf.WriteString(fmt.Sprintf("`%s`", alias))
+		}
+		buf.WriteString("\n")
+	}
```

Comment:
Please note this could be simpler, and faster


```suggestion
		buf.WriteString("Aliases: `")
		for i, alias := range cmd.Aliases {
			if i > 0 {
				buf.WriteString("`, `")
			}
			buf.WriteString(alias)
		}
		buf.WriteString("`\n")
	}
```